### PR TITLE
Add third commit

### PR DESCRIPTION
**Pull request by Ken-Yokohama**
I have added a third commit hehe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 This is a practice repository to learn how to contribute
 
 This is my second commit containing some errors. Hopefully someone can fix it!
+
+This is my third commit ;)
```
Comment (markbalechy) [inline README.md]: I don't like the winky face

File: README.md [hunk 2]
```diff
@@ -2,3 +2,5 @@
 This is a practice repository to learn how to contribute
```
Comment (markbalechy) [inline README.md]: Lets remove this extra line over here

**Review by markbalechy — CHANGES_REQUESTED** [review]
A few changes

File: README.md (hunk 1 above)
Comment (Ken-Yokohama) [reply README.md]: Copy that, removing winky face

File: README.md (hunk 2 above)
Comment (Ken-Yokohama) [reply README.md]: I agree, taking care of it now

**Review by markbalechy — APPROVED** [review]
LGTM!

**Pull request merged**